config/manifests/bases/update-service-operator.clusterserviceversion.yaml

```diff
@@ -66,11 +66,11 @@ spec:
   installModes:
   - supported: true
     type: OwnNamespace
-  - supported: true
+  - supported: false
     type: SingleNamespace
   - supported: false
     type: MultiNamespace
-  - supported: true
+  - supported: false
```

Comment:
What does this do for folks who are updating from a previous release that they'd installed in `AllNamespaces` mode?

Reply:
The operator doesn't support seamless upgrades so it may be a matter of uninstalling the current operator and then installing the later version at which time it will be bounded by its restrictions. When AllNamespaces install mode is selected installation is limited to `openshift-operators` and `openshift-update-service`. When AllNamespaces install mode is not supported the operator will not install successfully into `openshift-operators`. I created [OTA-506](https://issues.redhat.com/browse/OTA-506) to document such things.